README.md

```diff
@@ -162,7 +157,7 @@ To extend the instrumentation with the OpenTelemetry Instrumentation for Java,
 you have to use a compatible API version.
 
 <!-- IMPORTANT: do not change comments or break those lines below -->
-The Splunk Distribution of OpenTelemetry Java version <!--SPLUNK_VERSION-->1.19.0<!--SPLUNK_VERSION--> is compatible
+The Splunk Distribution of OpenTelemetry Java version <!--SPLUNK_VERSION-->1.20.0<!--SPLUNK_VERSION--> is compatible
 with:
 
 * OpenTelemetry API version <!--OTEL_VERSION-->1.21.0<!--OTEL_VERSION-->
```

Comment:
Hold on, did you run the pre-release script with all options filled? (`--otel`, `--otel-instrumentation`)
It should bump these versions too

Reply:
Nope. I'll do that and get the instructions updated.